src/library_idbfs.js

```diff
@@ -239,7 +239,6 @@ mergeInto(LibraryManager.library, {
 
       var remove = [];
       Object.keys(dst.entries).forEach(function (key) {
-        var e = dst.entries[key];
         var e2 = src.entries[key];
```

Comment:
While you are there maybe eliminate e2 too.. especially since it would otherwise have a particularly useless name.  How about just `if (!src.entries[key])`?